scripts/core/data/users/index.tsx

```diff
@@ -26,7 +26,7 @@ export function users(state = initialState, action: IUserAction): IUserState {
     }
 
     case 'UPDATE_ENTITY':
-        if (action.payload.resource === 'users') {
+        if (action.payload.resource === 'users' && state.entities[action.payload._id] !== action.payload.data) {
```

Comment:
When would this action fire with old data? Is there any use in comparing object references rather than data?

Reply:
it's related to the [cache PR](https://github.com/superdesk/superdesk-client-core/pull/3892), if there are multiple events triggering updates of the same user it might get called with same object when it gets resolved, so this will just update the state 1x instead of multiple times

Reply:
Could you handle it earlier before it reaches the reducer? It'd be best to debounce the callback that is triggered by a websocket event so we aren't dispatching actions that might get discarded.